From 9d9d001a1cd9265dab0b7ea20e87a734c08bc216 Mon Sep 17 00:00:00 2001
From: Simon Horman <horms+renesas@verge.net.au>
Date: Wed, 3 Jul 2013 13:38:29 +0900
Subject: [PATCH 0966/1083] Revert "drm/rcar-du: Fix planes allocation for
 multiplanar formats"

This reverts commit 90864406694083d5b77505204d8f8a33f0713a7a.
(cherry picked from commit 9b910368e8d02692d81bdb6180c5b1f6e2178903)

Signed-off-by: Ryo Kataoka <ryo.kataoka.wt@renesas.com>
---
 drivers/gpu/drm/rcar-du/rcar_du_plane.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/gpu/drm/rcar-du/rcar_du_plane.c b/drivers/gpu/drm/rcar-du/rcar_du_plane.c
index 94bd0e2..220ca00 100644
--- a/drivers/gpu/drm/rcar-du/rcar_du_plane.c
+++ b/drivers/gpu/drm/rcar-du/rcar_du_plane.c
@@ -49,6 +49,8 @@ int rcar_du_plane_reserve(struct rcar_du_plane *plane)
 		if (plane->format->planes == 1 ||
 		    rcdu->planes.free & (1 << ((i + 1) % 8)))
 			break;
+
+		break;
 	}
 
 	if (i == ARRAY_SIZE(rcdu->planes.planes))
-- 
1.8.3.2

